src/sentry/migrations/0665_add_active_monitor_incident_index.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 5.0.2 on 2024-03-06 19:53
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
```

Comment:
This table is large enough that adding this index will probably time out during deploy. Let's set this to dangerous. You can then run it yourself: https://www.notion.so/sentry/Running-Dangerous-Migrations-859a553040fc4649b1f8dc27055c8619?pvs=4#39e52a7d48a64895bf9287dc1f4af581

Reply:
dang good call, 3M+ rows on this table 0.0